pkg/transport/proxy/transparent/transparent_proxy.go

```diff
@@ -364,7 +364,8 @@ func (p *TransparentProxy) Start(ctx context.Context) error {
 	var finalHandler http.Handler = handler
 	for i := len(p.middlewares) - 1; i >= 0; i-- {
 		finalHandler = p.middlewares[i](finalHandler)
-		logger.Infof("Applied middleware %d\n", i+1)
+		// TODO: we should really log the middleware name here
```

Comment:
why haven't you done it now?

Reply:
I didn't see an easy way of doing it and I was in the midst of the OTEL work when I put this TODO. Let me see if I can do it now.